pallets/gear/src/tests.rs

```diff
@@ -16730,4 +17332,96 @@ pub(crate) mod utils {
             })
             .collect()
     }
+
```

Comment:
most of the code of this "snapshot" seems useless within gear pallet, so please rework the test logic - you only need to check pages as far as I see, any checks for program active status and queue may be done directly from gear pallet or even removed as redundant checks